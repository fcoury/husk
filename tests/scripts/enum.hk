enum Color {
  Red,
  Green,
  Blue,
}

let color = Color::Green;
println!(color);
