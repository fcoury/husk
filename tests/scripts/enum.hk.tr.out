Color { variant: 'Green', value: undefined }
